### ISSUE TITLE
Investigate switching to a Mozilla graphing library

### ISSUE THREAD

**Issue by Mark-Simulacrum** [rust-lang-nursery/rustc-perf#91]
This https://github.com/mozilla/metrics-graphics library by mozilla looks well maintained and relatively good looking.

Marking as low priority though since we're mostly happy with HighCharts (though @nrc did mention that it looks worse in his opinion).

**Comment by Mark-Simulacrum** [comment]
Upon further investigation, the library depends upon D3 and jQuery, this is a bit much... closing.
